utils/src/lib.rs

```diff
@@ -4,8 +4,8 @@ use bytes::{BufMut, BytesMut};
 use commonware_codec::{EncodeSize, Write};
 use std::time::Duration;
 
-pub mod array;
-pub use array::Array;
+pub mod sequence;
```

Comment:
I don't know if I love the name `sequence`, might be an overloaded term...

But I don't have anything much better